src/software/world/field.h

```diff
@@ -39,21 +39,23 @@ class Field
 
     /**
      * Constructs a new field with the given dimensions
-     * Note: All dimensions must be positive
+     *
+     * @pre all dimensions must be > 0
```

Comment:
including the buffer? just curious but I can (logically) see that being zero

Reply:
Great point, I've changed it to allow boundary buffers of `0`